tests/stdlib/tstats.nim

```diff
@@ -1,46 +1,55 @@
-include stats
+import std/stats
 
-proc clean(x: float): float =
-  result = round(1.0e8*x).float * 1.0e-8
+proc `~=`(x, y: float): bool =
+  abs(x - y) < 10e-8
```

Comment:
would math.almostEqual work?

Reply:
Not without adjusting the `unitsInLastPlace` parameter, I tried using `almostEquals` first.

Reply:
> Not without adjusting the unitsInLastPlace parameter

adjusting that param would be fine